api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -52,7 +51,7 @@
 	 */
 	public CohortDefinition getPatientsWhoAreLessThan3MonthsOnArt() {
 		CalculationCohortDefinition cd = new CalculationCohortDefinition("On ART for less than 3 months",
-		        new OnArtForLessThanXmonthsCalcultion());
+		        Context.getRegisteredComponents(OnArtForLessThanXmonthsCalcultion.class).get(0));
```

Comment:
My evaluator will need new objects(instances) of the calculation being created, why would we want to go through the Context for this? any benefits for this that the previous method?

Reply:
I don't think we need to  be instantiating calculation classes to the evaluator, the context will provide one instance of the OnArtForLessThanXmonthsCalcultion